### DISCUSSION TITLE
Could not retrieve maps

### DISCUSSION THREAD

**Discussion by blackhawk381** [microsoft/Dual-write-automations#24] [Q&A]
Hi,

We have copied environments and on the copy I would like to start the DW
so I exported the original environment and when trying to import the DW, I keep getting "Could not retrieve maps, retrying in some time"
I can login (and have access to DW), and the codebase is the same, so I was wondering what the issue could be.

**Comment by ollitheorca** [comment]
Hi @blackhawk381,

Does the application stop running, or is this just being logged while it is running?

I have had the same issue while the application is running. But usually, when I let the application run it will get the maps in the end, and then it will finish the config.

If the application stops due to this could you then try to share the log file?

**Comment by blackhawk381** [comment]
It doesn't crash the app. 
haven't tried letting it run for a long time.
I'll try this and will come back to you.   (could take awhile since I started manually now and we're having some other issues)

**Reply by Hax1337** [reply]
@blackhawk381 reaching out, can you try with the latest release?

**Reply by blackhawk381** [reply]
regretfully my system administration team blocked the use of Anonymous browsers to login, 
so for the moment I can't use the app because of the login in a private Edge browser.

**Reply by Hax1337** [reply]
@blackhawk381 with the latest version you can disable private browsing, this weill then use the default browser instead:
![image](https://github.com/microsoft/Dual-write-automations/assets/33001436/57b868e8-8dc9-499b-b1bd-8fc3d67454ca)

**Reply by blackhawk381** [reply]
Retried it, 
first test was semi-successful, 
Dual-write itself was mostly done, but didn't configure the integration keys correctly
because of that some tables failed, but the first issue seems ok now

**Reply by Hax1337** [reply]
Is this resolved @blackhawk381 ?

**Comment by blackhawk381** [comment]
Hi

We have not encountered this issue again in the newer versions

Op do 18 jul 2024 om 15:06 schreef Max Hentschel ***@***.***>:
